### PR TITLE
Skadnetlist

### PR DESCRIPTION
**Pull request by bszekely1**

### CODE REVIEW

File: extensions/community_extensions/skadnetwork.md [hunk 1]
```diff
@@ -32,12 +44,14 @@ The responsibilities of each participant when using the SKAdNetwork specificatio
...
 2. Support OpenRTB extension objects: `BidRequest.imp.ext.skadn` & `BidResponse.imp.ext.skadn`
-3. Return all necessary signed parameters to SSP/SDK to facilitate ad signatures and receive install validation postbacks at endpoint established during SKAdNetwork registration with Apple
+3. Determine at runtime if their entity is in the bid stream either by the included raw list and / or within the IABTL max list, or excluded from said list
+4. Return all necessary signed parameters to SSP/SDK to facilitate ad signatures and receive install validation postbacks at endpoint established during SKAdNetwork registration with Apple
 
 #### Publishers/source app’s responsibilities are to:
 
 1. Add the ad network’s ID to its Info.plist
 2. Update Info.plist with new entries added to the SSP/SDK publicly hosted lists when publishing new app versions to the App Store
+3. Supply the raw `skadnetids`, IABTL `max` and / or `exlc` to the SSP / SDK on the device at runtime
```
Comment (simontrasler) [inline extensions/community_extensions/skadnetwork.md:54]: Typo, `excl`.

File: extensions/community_extensions/skadnetwork.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # SKAdNetwork
 
-Sponsors: MoPub, Fyber
+Sponsors: MoPub, Fyber, Mangite (formerly Rubicon Project)
```
Comment (simontrasler) [inline extensions/community_extensions/skadnetwork.md:3]: Typo, Magnite.

File: extensions/community_extensions/skadnetwork.md [hunk 3]
```diff
@@ -426,6 +538,49 @@ DSPs may also want to understand what is the status of a user on iOS 14+. The `a
 * "DNT" or "LMT" = 1 when "ATTS" = 0, 1, 2
 * "LMT" or "DNT" = 0 when "ATTS" = 3
 
+
+## IABTL managed SKAdnetwork ID list
+
+The IABTL managed list of SKAdNetwork IDs is to address the communition large lists across complex programmatic supply chains where the use of `skadnetids` is not feasible.
```
Comment (reshmin) [inline extensions/community_extensions/skadnetwork.md:544]: Typo "communition"

**Comment by amitshetty** [comment]
On the IAB Tech Lab responsibilities, we should change "Not validate any input data" to 
"Will not validate the ID with Apple, but will verify that the domain matches the domain of the verified email address of the submitter. Will also provide an offline/non-autmoated process in case the email domain is different."

**Comment by amitshetty** [comment]
nonlistskadnetids - why do we need this when we already have skadnetids? (I'm guessing this got discussed on the last call - my apologies, but would be good to state why somewhere). 

Also, do we need to reduce the length of the attribute name? :)

File: extensions/community_extensions/skadnetwork.md [hunk 4]
```diff
@@ -32,12 +44,14 @@ The responsibilities of each participant when using the SKAdNetwork specificatio
 
 1. Provide SKAdNetwork IDs to each supply partner
 2. Support OpenRTB extension objects: `BidRequest.imp.ext.skadn` & `BidResponse.imp.ext.skadn`
-3. Return all necessary signed parameters to SSP/SDK to facilitate ad signatures and receive install validation postbacks at endpoint established during SKAdNetwork registration with Apple
+3. Determine at runtime if their entity is eligible for attribution postbacks
```
Comment (jeffreycarlson) [inline extensions/community_extensions/skadnetwork.md:47]: "at runtime" is a bit specific in implementation, would suggest we remove to be more flexible.

Suggested: "Determine if their entity is eligible for attribution postbacks"

File: extensions/community_extensions/skadnetwork.md [hunk 5]
```diff
@@ -100,7 +114,7 @@ If a DSP has at least one SKAdNetworkItem in the publisher app’s `Info.plist`
         <code>skadnetids</code>
       </td>
       <td>
-        A subset of SKAdNetworkItem entries in the publisher app’s Info.plist that are relevant to the DSP. Recommended that this list not exceed 10.
+        A subset of SKAdNetworkItem entries in the publisher app’s Info.plist that are relevant to the bid request. Recommended that this list not exceed 10. Note: `skadnetids` will eventually be deprecated in favor of `BidRequest.imp.ext.skadn.skadnetlist.nonlistskadnetids`.
```
Comment (jeffreycarlson) [inline extensions/community_extensions/skadnetwork.md:117]: I would remove this edit per previous feedback. There is a market use case for `skadnetids` even in the full list solution.

File: extensions/community_extensions/skadnetwork.md [hunk 6]
```diff
@@ -109,6 +123,23 @@ If a DSP has at least one SKAdNetworkItem in the publisher app’s `Info.plist`
...
+      <td>
+        object containing the IABTL list definition
+      </td>
+      <td>
+        object
+      </td>
+      <td class="text-monospace">
+        “skadnetlist":{
+          “max":306,
+          "excl":[2,8,10,55]
```
Comment (jeffreycarlson) [inline extensions/community_extensions/skadnetwork.md:139]: Add the JSON type  quotes here.

Comment (bszekely1) [reply extensions/community_extensions/skadnetwork.md:139]: cleaned up all the incorrect quotes.

File: extensions/community_extensions/skadnetwork.md [hunk 7]
```diff
@@ -11,17 +12,25 @@ The following are the updates provided in this document
...
 
 ## SKAdNetwork Extension
 
 ### Participant responsibilities
 
 The responsibilities of each participant when using the SKAdNetwork specifications are as follows.
 
+#### IABTL responsibilities are to:
+1. Organize SKAdNetwork IDs as well as define an automated self-serve registration process
+    - Will not validate the ID with Apple, but will verify that the domain matches the domain of the verified email address of the submitter. Will also provide an offline/non-autmoated process in case the email domain is different.
```
Comment (olancheg) [inline extensions/community_extensions/skadnetwork.md:26]: > non-autmoated

typo

**Pull request merged**